docs/dev_guide/contents/pr_review_procedure.rst

```diff
@@ -25,11 +25,19 @@ Before the "merge" button is clicked the following criteria must be met:
 
 * All the continuous integration must pass unless there is a known issue.
 
-* At least two members (not the author of the PR) of the "sunpy-developers" group must have approved the PR, one should, ideally, be a relevant subpackage maintainer.
+* At least two members (excluding the PR author) of the "sunpy-developers" group must have approved the PR.
+  Exceptions can be made for minor changes, see below.
 
 * All comments posted on the thread must be resolved.
 
-It is important that approval for merging the PR is done on the comment thread, as this becomes part of the "permanent record", this includes in during community meetings or in chat.
+It is important that approval for merging the PR is always done by explicitly approving the PR through the GitHub UI before merging, so a record of approval is left in the PR thread.
+
+Minor changes
+-------------
+
+If a PR only makes minor changes, it can be merged by the first reviewer, if they are confident they fully understand the changes.
```

Comment:
How are we defining "minor changes" here? A bugfix? A documentation rewording? In the past, we've said bugfixes only require a single review. In my mind, a bugfix is more clearly defined than a "minor change." Here, we seem to be broadening the scope, but not defining what that scope is. I don't disagree with the spirit of this, but o avoid just having this same debate on every single PR that someone marks as "minor changes" I think we should make an attempt here to define what a "minor change" actually means.

Reply:
What metrics could we use? 

I can think of:
- Line count of changes, say less than 10?
- The location of the change, e.,g a CI file vs a Python module
- Others?

Reply:
I'd be reluctant to have a hard limit on lines. Also reluctant to have hard rules in general, I think we all trust each others judgement when it comes to reviewing? Agree that some guidelines would definitely be helpful though, perhaps one-reviewer PRs could be:

- Improvements to existing documentation (including adding missing docstrings)
- 'Minor' bugfixes (including to the CI), leaving open interpretation of 'minor'
- **Not** new features (including to the CI), no matter how 'minor'
- **Not** new documentation pages (e.g. a new example, tutorial, page)
?

Reply:
That sounds good to me. Anything new does not count as minor

Reply:
This should simply just be explained by example then, something like:

Exactly what constitutes a minor-change is left up the the reviewer but some examples might include:

- Improvements to existing documentation (including adding missing docstrings)
- 'Minor' bugfixes (including to the CI), leaving open interpretation of 'minor'
- Changes in code style rather than substance

A minor-change can never include:

- New features (including to the CI), no matter how 'minor'
- New documentation pages (e.g. a new example, tutorial, page)
- API changes
- etc.

Reply:
Agree about not having hard rules and trusting each other's judgement. I think enumerating examples of minor changes is sufficient as Alasdair has done above. I would exclude "etc." though and amend API changes to "Changes to the public-facing API". We can always amend this list as needed if at some point we feel it needs more clarification.

Reply:
I still think the CI should be special cased.